docs/data-sources/dis_partition.md

```diff
@@ -4,15 +4,17 @@ subcategory: "Data Ingestion Service (DIS)"
 
 # huaweicloud_dis_partition
```

Comment:
suggest to add a new `huaweicloud_dis_partitions` as an alternative to `huaweicloud_dis_partition`